src/main/resources/alma/alma.fix

```diff
@@ -15,6 +15,11 @@ de.hbz.lobid.helper.DropLocal()
 
 add_field("@context","http://lobid.org/resources/context.jsonld")
 
+do list(path:"880??","var":"$i")
+  copy_field("$i.6","$i.linkageTest")
+  replace_all("$i.linkageTest","\\d{3}(-\\d{2}).*","880$1")
+end
```

Comment:
What is this supposed to do?

Reply:
This prepares the comparison in the linkage field and allows it to be angostic about the field name:

https://github.com/hbz/lobid-resources/blob/57f3f7916ae7c4f3930f2efd9672600339ea986f/src/main/resources/alma/fix/macros.fix#L4-L9

Reply:
ignore the outcomment line it can be deleted.

Reply:
I see, thanks.